images/users/Icicletech-logo.svg

```diff
@@ -0,0 +1,721 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="500px" height="200px" viewBox="0 0 500 200" enable-background="new 0 0 500 200" xml:space="preserve">  <image id="image0" width="500" height="200" x="0" y="0"
+    xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAfQAAADICAYAAAAeGRPoAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
```

Comment:
It's a PNG inlined into a SVG. This might be configurable during the export of the logo, see other logos https://raw.githubusercontent.com/babel/website/master/website/static/img/users/n26.svg which are using real svg paths.

Reply:
Thanks. Let me get the right format and get back to you.